src/EmbeddedVsix/EmbeddedVsix.csproj

```diff
@@ -0,0 +1,115 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
+    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
+    <TargetFrameworkProfile />
+  </PropertyGroup>
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectTypeGuids>{82b43b9b-a64c-4715-b499-d71e9ca2bd60};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <ProjectGuid>{4BBE7B39-E00A-40FE-88FC-A53C7E49731F}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>SonarLint.VisualStudio.AdditionalFiles</RootNamespace>
```

Comment:
The name AdditionalFiles is slightly roslyn-like, I think something like "Analyzers" or "ExternalAnalyzers" is a bit more appropriate, but it doesn't matter much. I don't know if it's worth changing.

Reply:
I thought about other names but wasn't happy with any of them. We have multiple analyzers and this VSIX only contains one of them so calling it "Analyzers" seems misleading. Ditto "ExternalAnalyzers" - we might be able to put the CFamily analyzer files in this VSIX, but that's hundreds of extra files so that seems risky. Also, it not clear what makes an analyzer "internal" or "external".
In the end I decided to name the VSIX based on its function, which is just as a way of packing any extra stuff that doesn't fit in the main package.